### ISSUE TITLE
Relation between silviomoreto/bootstrap-select and snapappointments/bootstrap-select is unclear

### ISSUE THREAD

**Issue by ddinchev** [snapappointments/bootstrap-select#1979]
There seems to be relation between this package and https://github.com/snapappointments/bootstrap-select. They follow the same versioning and have the same bower name. Neither of the packages however states how they are related and there is a mess with bower. For me, `bower info bootstrap-select` resolves the `snapappointments/bootstrap-select` package.

```
➜  ~ bower info bootstrap-select
bower bootstrap-select#*    not-cached https://github.com/snapappointments/bootstrap-select.git#*
bower bootstrap-select#*       resolve https://github.com/snapappointments/bootstrap-select.git#*
bower bootstrap-select#*      download https://github.com/snapappointments/bootstrap-select/archive/v1.13.1.tar.gz
bower bootstrap-select#*       extract archive.tar.gz
bower bootstrap-select#*      resolved https://github.com/snapappointments/bootstrap-select.git#1.13.1

...
```

On the other hand:
`composer require bootstrap-select/bootstrap-select` does resolve `silviomareto/bootstrap-select` but only up to version `1.12.4` as there hasn't been any new release tags for a year.

Can you clearly state how are the packages related? Which should be installed and what would be the right way to do so?

**Comment by Jogai** [comment]
Also, see https://github.com/silviomoreto/bootstrap-select/issues/1915#issuecomment-383635185

**Comment by caseyjhol** [comment]
After actively developing bootstrap-select for over 5 and a half years and writing close to 30,000 lines of code, I wanted full control of the project to ensure I could take it in the right direction. I also wanted the ability to add collaborators and update the project's description, as well as adjust other repository settings without having to always contact Silvio, thus speeding up development.

Seeing as Silvio hasn't shown much interest in the project the past 5 years (he hasn't written a line of code since 2013), I was hoping he would be willing to transfer ownership to me to simplify things and ease the transition for the community. He has his reasons for not wanting to do that, though, so I opted to fork the original repository and continue my development there.

https://github.com/snapappointments/bootstrap-select-temp/issues/12#issuecomment-382790871

I opted not to transfer the Composer package, as the naming scheme was inconsistent. The correct composer package is [snapappointments/bootstrap-select](https://packagist.org/packages/snapappointments/bootstrap-select).
